rollup/core/objects/RollupLoggerPluginParameter__mdt/fields/Description__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Description__c</fullName>
+    <description>Add information pertaining to the parameter value here, if necessary</description>
+    <externalId>false</externalId>
+    <fieldManageability>SubscriberControlled</fieldManageability>
```

Comment:
For the description fields in Nebula's plugin/plugin-parameter objects, I marked the fields as `DeveloperControlled` ([example](https://github.com/jongpie/NebulaLogger/blob/main/nebula-logger/main/plugin-framework/objects/LoggerSObjectHandlerParameter__mdt/fields/Description__c.field-meta.xml#L5)) so that I can update the description text in future package versions. With `SubscriberControlled`, I think you would only be able to set it on the first install for an org, so you lose the ability to later update the description text.

Reply:
thank you, I will update this as well!